src/index.html

```diff
@@ -5,9 +5,40 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin=" ">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="imac"
+      class="card__image"
+      >
```

Comment:
Add 2 spaces for attributes and check it everywhere
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="imac"
        class="card__image"
      >
```